### ISSUE TITLE
Public Transit Services Run By Cities

### ISSUE THREAD

**Issue by Wuzzle** [OpenLoco/OpenLoco#2664]
**Describe the solution you'd like**
Cities would run their own public transit services once they grow past a certain point and if they haven't already been sufficiently covered by you or a competitor's services. Ideally you'd be able to buy out these services.

**Additional context**
The lack of normal traffic and non-company owned public transit services has always made the game feel lifeless and empty and I think this would be a step in the right direction towards fixing that.

**Comment by LeftofZen** [comment]
This sounds like you want an AI company to build services? We've always talked about wanting to make the AI scriptable, so people can write their own AI companies. One thing you'd be able to do with scriptable AI is implement something exactly like this - an AI that localises around a city. That's a very far off feature though and we don't even have a feature request for it that I can find.

A mechanic to 'buy out' competitors like in OpenTTD is possible, but not something we've talked about. I'm not so sure about the 'city simulation' aspect though as this game is primarily a transport/train simulation game, not a city-builder like Cities Skylines, Transport Fever, etc. It would be possible to have some purely visual elements to liven aspects of the game up though

**Comment by Wuzzle** [comment]
Yeah that'd be fun. I was thinking you could do that from a code perspective to simulate a town council

As for city simming stuff, don't worry about it, there's already a lot of imagination being used for this game
